bower.json

```diff
@@ -21,6 +21,6 @@
         "spec"
     ],
     "dependencies": {
-        "moment": "~2.10.3"
+        "moment": "~2.12"
```

Comment:
On this note, if it's not necessary to have the `build` directory any longer (with the minified JS), I'd like to drop it.  Bower expertise would be appreciated on this question.